### PR TITLE
WIP: [CFM-1554-fix] Align c4m_body field configuration of photo CT, to match other CTs.

### PR DESCRIPTION
**Pull request by anvmn**
https://issuetracker.amplexor.com/jira/browse/CFM-1554

1. Avoid using Mail format.
2. Set formats order to : 
   -  'default_html', 
   -  'filtered_html',
   -  'full_html',
   -  'plain_text.

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_photo/c4m_content_photo.features.field_instance.inc [hunk 1]
```diff
@@ -244,6 +282,81 @@ function c4m_content_photo_field_default_field_instances() {
     ),
   );
 
+  // Exported field_instance: 'node-photo-c4m_created_with_photo_album'.
+  $field_instances['node-photo-c4m_created_with_photo_album'] = array(
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_photo/c4m_content_photo.features.field_instance.inc]: how is this field related ?

**Comment by anvmn** [comment]
Closing in favor of #1521

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_photo/c4m_content_photo.features.field_instance.inc (hunk 1 above)
Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_photo/c4m_content_photo.features.field_instance.inc]: Not related...
I removed in at #1521 that replaces this PR
